src/baseline/riscv32/baseline-assembler-riscv32-inl.h

```diff
@@ -418,9 +418,9 @@ void BaselineAssembler::AddToInterruptBudgetAndJumpIfNotExceeded(
 void BaselineAssembler::AddSmi(Register lhs, Smi rhs) {
   ASM_CODE_COMMENT(masm_);
   if (SmiValuesAre31Bits()) {
-    __ Add32(lhs, lhs, Operand(rhs));
```

Comment:
So what's the difference of the if and else? Should merge or deal seperately?

Reply:
done